### ISSUE TITLE
ps roi align

### ISSUE THREAD

**Issue by DayBreak-u** [leowangzi/LightHeadRCNN#1]
hi，Thanks for your contributing！Do you try psroipool or psroialign

**Comment by ouyanghuiyu** [comment]
cls_score = self.RCNN_cls_score(pooled_feat)
        cls_prob = F.softmax(cls_score, 1)

        RCNN_loss_cls = 0
        RCNN_loss_bbox = 0

        if self.training:
            # classification loss
            RCNN_loss_cls = F.cross_entropy(cls_score, rois_label)

这个loss应该算错了吧，应该是F.cross_entropy(cls_prob, rois_label)
